cs/src/Management/TunnelManagementClient.cs

```diff
@@ -921,13 +921,19 @@ public void Dispose()
             Requires.NotNull(endpoint, nameof(endpoint));
             Requires.NotNullOrEmpty(endpoint.HostId!, nameof(TunnelEndpoint.HostId));
 
+            string? query = null;
+            if (tunnel.Ports != null && tunnel.Ports.Where((p) => p.Protocol == TunnelProtocol.Ssh).Any())
+            {
+                query = "includeSshGatewayPublicKey=true";
```

Comment:
Can `includeSshGatewayPublicKey` be a string constant in a contract? Then tunnel service may also use that constant.